Ed-Fi-Exchange-OSS/EdFi-Project-Buzz
# [FIF-286] Move React components to EdFi.Buzz.UI - Admin surveys

Add the survey admin page

# Testing
- Should show the survey list
- Delete button should work
- Update button should redirect to /uploadSurvey/{surveyId}
